code/atom.dm

```diff
@@ -763,30 +771,19 @@
 
 	return null
 
-/atom/MouseDrop(atom/over_object as mob|obj|turf)
-	SPAWN( 0 )
-		if (istype(over_object, /atom))
-			if (isalive(usr))
-				//To stop ghostdrones dragging people anywhere
-				if (isghostdrone(usr) && ismob(src) && src != usr)
-					return
-
-				/* This was SUPPOSED to make the innerItem of items act on the mousedrop instead but it doesnt work for no reason
-				if (isitem(src))
-					var/obj/item/W = src
-					if (W.useInnerItem && W.contents.len > 0)
-						target = pick(W.contents)
-				//world.log << "calling mousedrop_t on [over_object] with params: [src], [usr]"
-				*/
-
-				over_object.MouseDrop_T(src, usr)
-			else
-				if (istype(over_object, /obj/machinery)) // For cyborg docking stations (Convair880).
-					var/obj/machinery/M = over_object
-					if (M.allow_stunned_dragndrop == 1)
-						M.MouseDrop_T(src, usr)
```

Comment:
This is one of the WTF? moments of the code. This allows dead mobs to cause MouseDrop_T but only if allow_stunned_dragndrop is true. Removing this doesn't affect cyborg docks. I wasn't around this far back but it might be because of some weird borg stuff from ancient times.

Reply:
this comment would be from like 8 years ago